lib/db/user.ts

```diff
@@ -24,6 +25,7 @@ export async function createUser(params: Omit<User, "id">): Promise<User> {
       ...params,
     },
   });
+  createActivityLog("Create", "user", user.id);
```

Comment:
This call throws an error since there is no user session before a user signs up -- going to move the call to after the user signs up.